Ginger/GingerCoreNET/SourceControl/GITSourceControl.cs

```diff
@@ -323,6 +323,11 @@ public override bool GetLatest(string path, ref string error, ref List<string> c
             }
             catch (Exception ex)
             {
+                if(ex is AggregateException && ex.InnerException is CheckoutConflictException)
+                {
+                    error = "Local branch has uncommited changes, check-in them before getting latest.";
```

Comment:
Please add this error in message dictionary and refer it from there.
Also add exception to log, in this case it will return without logging the error.

Reply:
Done